app/models/card.rb

```diff
@@ -1,7 +1,16 @@
 class Card < ActiveRecord::Base
   before_create :set_review
+  validate :check_card
 
-  def set_review
-    self.review = Time.now + 3.days
-  end
+  protected
+    def set_review
+      self.review = Time.now + 3.days
+    end
+
+    def check_card
+      self.original = original.downcase
+      self.translated = translated.downcase
+      errors.add(:original, ' = Translated. Вводимые значения должны отличаться.') if
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.